libs/constrained-generators/constrained-generators.cabal

```diff
@@ -39,6 +40,10 @@ library
     Constrained.Generic
     Constrained.Graph
     Constrained.List
+    Constrained.Min.Base
```

Comment:
`Min` is a terrible suffix. Is it related to minimum value or the opposite of maximum?

Since it is only related to the manual, I'd recommend calling it something along the lines of "Manual"

Reply:
So its going to be moved to   Test.Minimal.Base
This is supposed to indicate it the Base module of the Minimal implementation, in the Tests